src/pages/guides/getting-started/deployment.mdx

```diff
@@ -4,23 +4,42 @@ export const description = 'How to deploy Nitric applications'
+You will need to configure your cloud credentials, either locally or in your CI/CD pipeline, to allow Nitric to create resources in a cloud account. Instructions on how to setup credentials for your target provider can be found in the provider documentation: [AWS](/reference/providers/aws), [Google Cloud](/reference/providers/gcp) and [Azure](/reference/providers/azure).
 
-### Configuring State Backend
+### Configuring state backend
 
-To maintain the state of your deployment between runs you'll need to configure a backend for Pulumi to store its stack state. For this, you can use either [Pulumi's managed service](https://www.pulumi.com/docs/concepts/state/#pulumi-cloud-backend), or you could use one of the other state [backends](https://www.pulumi.com/docs/concepts/state/#using-a-self-managed-backend) they provide support for.
+To track the state of your stack between deployments you'll need to configure a location for Pulumi to store its stack state (also known as a 'backend'). Pulumi provides several options for state backends, a common choice is [Pulumi's managed service](https://www.pulumi.com/docs/concepts/state/#pulumi-cloud-backend) which stores state in the cloud allowing for secure storage and collaboration. Alternatively, you can use any other state [backends](https://www.pulumi.com/docs/concepts/state/#using-a-self-managed-backend) Pulumi support.
 
-## Configuring Deployment
+<Note>
+  If you're trying Nitric for the first time we recommend starting with a
+  `local` backend, which stores the stack state in a file on your machine. If
+  you haven't already configured a backend for Pulumi, Nitric will prompt you to
+  automatically setup this local backend when you run `nitric up`.
+</Note>
+
+## Configuring stacks
+
+When using stacks to deploy to different environments such as a development, testing or production environment, you'll typically want to go beyond the default configuration Nitric uses when deploying cloud resources. This environment specific configuration is added to the stack files.
 
-When deploying the application, you can add extra configuration for specific handlers or a globally for all handlers. This configuration is unique to each stack file, allowing you to deploy the same application to different environments with different configuration.
+When deploying the application, you can add extra configuration for specific handlers or a globally for all handlers. This configuration is unique to each stack file, allowing you to deploy the same application to different environments with different configuration. Since the configuration options available are unique to the cloud platform and the services they offer these settings are typically different between provider implementations.
```

Comment:
```suggestion
When deploying the application, you can add extra configuration for specific handlers or globally for all handlers. This configuration is unique to each stack file, allowing you to deploy the same application to different environments with different configurations. Since the configuration options available are unique to the cloud platform and the services they offer these settings are typically different between provider implementations.
```